tests/test_cadquery.py

```diff
@@ -2771,7 +2770,7 @@ def testMakeShellSolid(self):
                 lines.append(
                     Edge.makeLine(Vector(*vertices[v1]), Vector(*vertices[v2]))
                 )
-            wire = Wire.combine(lines)
+            wire = Wire.combine(lines)[0]
```

Comment:
Why is only the first wire used here after the combine?

Reply:
I changed Wire.combine to return a list of Wires. Now it handles disconnected edges/wires properly, but obviously such a modification is needed.